geode-docs/tools_modules/geode_for_redis.html.md.erb

```diff
@@ -35,13 +35,14 @@ Use gfsh to start at least one server with a command of the form:
 start server \
   --name=<serverName> \
   --locators=<locatorPort> \
-  --geode-for-redis-port=<geodeForRedisPort> \
-  --geode-for-redis-bind-address=<geodeForRedisBindAddress> \
-  --geode-for-redis-password=<geodeForRedisPassword>
+  --J=-Dgemfire.geode-for-redis-enabled=true \
+  --J=-Dgemfire.geode-for-redis-port=<geodeForRedisPort> \
+  --J=-Dgemfire.geode-for-redis-bind-address=<geodeForRedisBindAddress> \
+  --J=-Dgemfire.geode-for-redis-password=<geodeForRedisPassword>
```

Comment:
There doesn't seem to be a `geode-for-redis-password` property in the codebase, so references to this should probably be removed from docs.

Reply:
Good point, we switched to using a SecurityManager!